Sources/DeepLearning/Layer.swift

```diff
@@ -1329,7 +1307,7 @@ public struct SimpleRNNCell<Scalar: TensorFlowFloatingPoint>: RNNCell, VectorNum
     }
 }
 
-/// An LSTM Cell.
+/// An LSTM cell.
```

Comment:
Shouldn't this be `A LSTM cell` ?

Reply:
Since people tend to pronounce the acronym direct, the pronunciation starts with a vowel, so "An" would be most appropriate.